cmd/transflect-operator/operator.go

```diff
@@ -346,6 +385,30 @@ func getConfig() (*rest.Config, error) {
 	return kubeCfg, nil
 }
 
+func (o *operator) newLock(id string) *resourcelock.LeaseLock {
+	return &resourcelock.LeaseLock{
+		LeaseMeta: metav1.ObjectMeta{
+			Name:      "lease-transflect",
```

Comment:
nit: the type of the resource is a lease, so i don't think it needs "lease" in the name. Perhaps `transflect-leader` might be better?

Reply:
ok, agreed.